src/components/AccountForms/LoginForm/index.js

```diff
@@ -1,7 +1,7 @@
-import React, { useState, useContext } from 'react';
+import React, { useState, useContext, useEffect } from 'react';
```

Comment:
useEffect no longer needed, I am triggering the the pageview google analytics event on browser history change, pls see app component